tests/test_config.py

```diff
@@ -13,9 +13,6 @@
 import pytest
```

Comment:
Review is **NOT** needed.
- Converting gen_with_app testing into pytest.mark.parametrize
- Replace deprecated function into pytest one